.github/workflows/deploy_agent.yml

```diff
@@ -121,6 +125,19 @@ jobs:
           }
         shell: pwsh
 
+  index-download-site:
+    name: Rebuild indexes on the download site
+    if: ${{ github.event.inputs.indexdownloadsite == 'true' }}
```

Comment:
This job is implicitly related to the deploy-download-site job. We don't check for that setting here, but I thing not checking that setting also makes sense if we want the flexibility to run the indexer without redeploying to the download site. I'm mostly calling this out to verify if that is the intention here.

Reply:
Yeah, my intention was to allow re-indexing without actually deploying. But then I realized that the `get-external-artifacts` job isn't conditional and would do a lot of throw-away work if all you wanted to do was update the indexes.

I could certainly update so that the index rebuilding is triggered off of the same parameter as the deploy download site job uses.